### DISCUSSION TITLE
划词翻译

### DISCUSSION THREAD

**Discussion by huawuhen** [immersive-translate/immersive-translate#377] [Ideas]
### 您希望的更新和改进是什么 | Update or Improve

可否加入划词翻译

**Comment by versun** [comment]
正在开发中

**Reply by ghost** [reply]
希望能像隔壁那样能有多个翻译
![图片](https://github.com/immersive-translate/immersive-translate/assets/11383988/e726118f-64bb-4548-adc7-ada7194d5569)
